tests/conftest.py

```diff
@@ -1,14 +1,49 @@
-from .helper import call_command
+import json
 import os
-import tempfile
-from .helper import spawn_background_process
-import sys
 import shutil
-import json
+import subprocess
+import sys
+import tempfile
+
 import pytest
 from dotenv import load_dotenv
 
-GL_ROOT_DIR = "test-data/gardenlinux"
+from .helper import call_command, spawn_background_process
+
+TEST_DATA_DIR = "test-data"
+GL_ROOT_DIR = f"{TEST_DATA_DIR}/gardenlinux"
+CERT_DIR = f"{TEST_DATA_DIR}/cert"
+
+
+def generate_test_certificates():
```

Comment:
Can we use Python native solutions to generate test certificates? I personally always appreciate a Pythonic source code :)

Reply:
using `cryptography` now to generate the cert